app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    original_file_name = split_command[1]
+    if "/" not in split_command[2]:
+        with (open(original_file_name, "r") as original_file,
+              open(split_command[2], "w") as renamed_file):
+            renamed_file.write(original_file.read())
+        os.remove(original_file_name)
+    else:
+        path_list = split_command[2].split("/")
+
+        for directory in range(1, len(path_list)):
+            if not os.path.exists("/".join(path_list[0:directory])):
+                os.mkdir("/".join(path_list[0:directory]))
```

Comment:
it can create directory structure if i need to create all folders, but in some test, i already have some folders, and i need to check is they exist and then create. When i`m trying to do makedirs(path) i have error, that some dir already exist
![image](https://github.com/mate-academy/py-move-file/assets/168102872/c81cbe04-4840-4f4a-bab7-b095a69a1c02)
![image](https://github.com/mate-academy/py-move-file/assets/168102872/e1aee7d4-9b7f-4c23-97a8-6a5d2392ace5)

Reply:
`exist_ok=True`